tests/testing_data/texts/duplicate_references.xml

```diff
@@ -69,24 +69,19 @@
                 <div type="textpart" subtype="book" n="1">
                     <div type="textpart" subtype="poem" n="pr">
                         <l n="1">Spero me secutum in libellis meis tale temperamen-</l>
-                        <l n="1">tum, ut de illis queri non possit quisquis de se bene </l>
                     </div>
                 </div>
                 <div type="textpart" subtype="book" n="1">
                     <div type="textpart" subtype="poem" n="2">
                         <l n="1">Spero me secutum in libellis meis tale temperamen-</l>
-                        <l n="1">tum, ut de illis queri non possit quisquis de se bene </l>
```

Comment:
Why are these line removed ?

Reply:
Removing these lines was accidental. And I thought that I had corrected that. I will add them back in.